src/test/debuginfo/function-arguments-naked.rs

```diff
@@ -34,7 +38,5 @@ fn main() {
 
 #[naked]
 fn naked(x: usize, y: usize) {
```

Comment:
```suggestion
extern "C" fn naked(x: usize, y: usize) {
```

Some calling conventions (e.g. `stdcall`) require a stack adjustment on return. Since the Rust calling convention is unspecified, it's better to explicitly use "C".